src/main/scala/firrtl/options/Phase.scala

```diff
@@ -73,9 +73,9 @@ trait DependencyAPI[A <: DependencyAPI[A]] { this: TransformLike[_] =>
   def dependents: Seq[Dependency] = Seq.empty
   private[options] lazy val _dependents: LinkedHashSet[Dependency] = new LinkedHashSet() ++ dependents.toSet
 
-  /** A function that, given a transform will return true if this transform invalidates/undos the effects of the input
-    * transform
-    * @note Can a [[firrtl.options.Phase Phase]] ever invalidate itself?
+  /** A function that, given *another* transform will return true if this transform invalidates/undos the effects of that
```

Comment:
Seems like to be extra clear *another* should refer to parameter a at least I think that is what *another* means.

Reply:
Nice catch. I'll fix this in a rebase.